app/src/main/java/io/gnosis/safe/ui/settings/app/GetInTouchFragment.kt

```diff
@@ -116,6 +116,7 @@ class GetInTouchViewModel
         val text = StringBuilder()
         text.appendln("${context.getString(R.string.app_name)} v${BuildConfig.VERSION_NAME}")
         text.appendln(context.getString(R.string.feedback_safe, activeSafe?.address?.asEthereumAddressChecksumString() ?: ""))
+        text.appendln(context.getString(R.string.feedback_chain, activeSafe?.chain?.name ?: ""))
```

Comment:
Is it possible to have more than one chain with the same name? If so, then we should include the chain id.

Reply:
Chain: <chain name> (<id>) ?